.clang-format

```diff
@@ -38,3 +38,5 @@ BraceWrapping:
 PointerAlignment: Right
 SortIncludes: false
 Cpp11BracedListStyle: false
+AlignConsecutiveMacros: AcrossEmptyLines
+PPIndentWidth: 4
```

Comment:
Shouldn't this be 1?

Reply:
I think you're correct.

I had this at the time since it matches the style that upstream uses, and clint seemed okay with it for certain places, but it wasn't consistent, especially in https://github.com/neovim/neovim/issues/17763. I decided to create a separate issue to track it, see https://github.com/neovim/neovim/issues/17763

Reply:
I actually had to revert that part again and use this instead https://github.com/kylo252/neovim/blob/d7cbb071218307722f5445374b54f4b47b63a4f1/.clang-format#L42